README.md

```diff
@@ -10,77 +9,69 @@
 
-# Installation
-This repository requires python3.8 or higher. To install, simply clone this repository and install the requirements.
-```bash
-git clone https://github.com/opentensor/bittensor-subnet-template.git
-cd bittensor-subnet-template
-python -m pip install -r requirements.txt
-python -m pip install -e .
-```
+When you are ready to write your own custom incentive mechanism, start with this starter template repository. It is preloaded with all the files you need to run a very simple incentive mechanism. This simple incentive mechanism rewards miners for responding with the multiple of the value sent by validators. 
 
-</div>
+This simple starter template is split into three primary files. To write your own incentive mechanism, you should edit these files. These files are:
+- `template/protocol.py`: Contains the definition of the wire-protocol used by miners and validators.
+- `neurons/miner.py`: Script that defines the miner's behavior, i.e., how the miner responds to requests from validators.
+- `neurons/validator.py`: This script defines the validator's behavior, i.e., how the validator requests information from the miners and determines the scores.
 
 ---
 
-Once you have installed this repo and attained your subnet via the instructions in the nested docs (staging, testing, or main) you can run the miner and validator with the following commands.
-```bash
-# To run the miner
-python -m neurons/miner.py 
-    --netuid <your netuid>  # Must be attained by following the instructions in the docs/running_on_*.md files
-    --subtensor.chain_endpoint <your chain url>  # Must be attained by following the instructions in the docs/running_on_*.md files
-    --wallet.name <your miner wallet> # Must be created using the bittensor-cli
-    --wallet.hotkey <your validator hotkey> # Must be created using the bittensor-cli
-    --logging.debug # Run in debug mode, alternatively --logging.trace for trace mode
-
-# To run the validator
-python -m neurons/validator.py 
-    --netuid <your netuid> # Must be attained by following the instructions in the docs/running_on_*.md files
-    --subtensor.chain_endpoint <your chain url> # Must be attained by following the instructions in the docs/running_on_*.md files
-    --wallet.name <your validator wallet>  # Must be created using the bittensor-cli
-    --wallet.hotkey <your validator hotkey> # Must be created using the bittensor-cli
-    --logging.debug # Run in debug mode, alternatively --logging.trace for trace mode
-```
+## Installation
 
-</div>
+### Before you proceed
+Before you proceed with the installation of the subnet, note the following: 
+
+- Following these instructions you can run your subnet locally for your development and testing, or on Bittensor testnet. We do **recommend** that you first run your subnet locally and finish your development and testing before running the subnet on Bittensor testnet. 
+- You can also run your subnet either as a subnet owner, or as a validator or as a miner. 
```

Comment:
There a number of entities here 

1. Person running the chain itself
2. Person creating the subnet (owner)
3. Person running the miner/s
4. Person running the validator/s

In practice on staging you will be all 4. On testing you wont run the chain yourself, you will be the owner but might also share the mining and validating with others. On main, you are most certainly not the only person running miners and validators.

Reply:
Yes. I will make a note of this while editing the `docs/*.md` files in my next PR (today). I will revisit this README after that, just to ensure consistency on this topic.